lib/object.gi

```diff
@@ -676,6 +676,29 @@ function( obj,str )
   TryNextMethod();
 end );
 
+#############################################################################
+##
+#V  TYPE_KERNEL_OBJECT  
+##
+##
+##
+##  TYPE_KERNEL_OBJECT is the type of data objects used internally in the 
+##  kernel which have no significant &GAP; callable methods and should not
+##  normally be seen at &GAP; level. These are typically lookup tables or
+##  buffers created and used within the kernel and containing binary data only
+##
+
+
+DeclareRepresentation( "IsKernelDataObjectRep", IsDataObjectRep, []);
+
+
+BIND_GLOBAL( "TYPE_KERNEL_OBJECT",
+          NewType(NewFamily("KernelObjectFamily", IsObject),
+          IsObject and IsKernelDataObjectRep));
+
+
+InstallMethod( String, [IsKernelDataObjectRep], o->"<kernel object>");
+
```

Comment:
The above also has several pointless empty lines, I'd not use more than 1 or 2 in a row (and right after a comment block, we normally have zero or one).